server/neptune/workflows/internal/deploy/terraform/runner.go

```diff
@@ -56,14 +58,23 @@ func (r *WorkflowRunner) Run(ctx workflow.Context, deploymentInfo DeploymentInfo
 			"atlantis_revision":   deploymentInfo.Revision,
 		},
 	})
-	terraformWorkflowRequest := terraform.Request{
-		Root:         deploymentInfo.Root,
+
+	request := terraform.Request{
 		Repo:         deploymentInfo.Repo,
+		Root:         deploymentInfo.Root,
 		DeploymentID: id.String(),
 		Revision:     deploymentInfo.Revision,
 	}
 
-	future := workflow.ExecuteChildWorkflow(ctx, r.Workflow, terraformWorkflowRequest)
+	// force plan reviews if we have diverged
```

Comment:
thoughts on moving this into `Deployer.Deploy()` where we evaluate the commit direction? That way, the `WorkflowRunner` does not need to know about the diffDirection.

Reply:
Feels weird to me to build the request outside of this though given we'd still want to pass the deployment info through, I get that it's weird to have commit direction be passed through but doesn't seem like there's a better solution.